crates/matrix-sdk/src/guaranteed_completion.rs

```diff
@@ -0,0 +1,263 @@
+// Copyright 2022 Famedly GmbH
+// Copyright 2022 The Matrix.org Foundation C.I.C.
```

Comment:
Good to know this is Famedly stuff 😄

Since this file was written entirely by you, the Matrix.org Foundation doesn't need to be / shouldn't be mentioned at all.

```suggestion
// Copyright 2022 Famedly GmbH
```

Reply:
@johannescpk tipped me off about that already :D I'll change it with the next commit